core/Plugin/Report.php

```diff
@@ -204,33 +189,14 @@ class Report
     protected $defaultSortOrderDesc = true;
 
     /**
-     * @var array
-     * @ignore
-     */
-    public static $orderOfReports = array(
```

Comment:
This was hard coded before and is now determined by `CategoryId` and `SubcategoryId`